src/main/scala/com/moesif/filter/MoesifApiFilter.scala

```diff
@@ -39,56 +36,18 @@ class MoesifApiFilter @Inject()(config: MoesifApiFilterConfig)(implicit mat: Mat
   private val eventModelBuffer = mutable.ArrayBuffer[EventModel]()
   private val client = new MoesifAPIClient(moesifApplicationId, moesifCollectorEndpoint)
   private val moesifApi = client.getAPI
-  private var lastTimeAppConfigFetched: DateTime = DateTime.parse("1970-01-01", DateTimeFormat.forPattern("yyyy-MM-dd"))
-  private var isAppConfigFetched: Boolean = false
-  private var appConfigModel = new AppConfigModel
 
-  val appConfigRunnable: Runnable = new Runnable() {
-    override def run(): Unit = {
-      getApplicationConfig()
-    }
-  }
   val eventBufferFlusher: Runnable = new Runnable() {
     override def run(): Unit = flushEventBuffer()
   }
   // Create an executor to fetch application config every 5 minutes
   val exec: ScheduledExecutorService = Executors.newScheduledThreadPool(1)
-  exec.scheduleAtFixedRate(appConfigRunnable, 0, 5, TimeUnit.MINUTES)
   // Initialize with a ScheduledFuture[_] that fires immediately, doing nothing
   private var scheduledSend: ScheduledFuture[_] = exec.schedule(eventBufferFlusher, 0, TimeUnit.MILLISECONDS)
```

Comment:
Will base api not take care of it?

Reply:
It's event schedule, I think we should keep this, @qmjava correct me if I am wrong

Reply:
we should keep it here.